test/test.js

```diff
@@ -840,7 +851,17 @@ test('should throw on unknown HTTP method', (t) => {
   t.plan(1)
   const dispatch = function (req, res) { }
 
-  t.throws(() => inject(dispatch, { method: 'UNKNOWN_METHOD', url: 'http://example.com:8080/hello' }), Error)
+  t.throws(() => inject(dispatch, { method: 'UNKNOWN_METHOD', url: 'http://example.com:8080/hello' }, (err, res) => {
+    t.ok(err)
+  }), Error)
+})
```

Comment:
Why this change?

Reply:
Because with the new api, if there is no callback function, it would be treated as using chainable apis (as shown in the next test).

Reply:
I believe that it would be better to add `.json()` handler of `res` in another PR, for it has no concern with chainable apis. Do you agree?